libs/models/src/lib/interfaces/logo-verify-response.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface LogoVerifyResponse {
+  status: boolean;
+  realImageUrl: string;
```

Comment:
I suppose my previous comment would then change this to `realImageUrl?: string`.